app/client/src/sagas/ErrorSagas.tsx

```
@@ -92,7 +92,8 @@ export function* errorSaga(
   } = errorAction;
   const message =
     error && error.message ? error.message : ActionErrorDisplayMap[type](error);
-  if (show && error && error.code !== 401) {
+  
+  if (show && error && error.show) {
```

The suppresses all errors which do not have the `{show: true}` property. Is this what we want?